docs/throttling.md

```diff
@@ -1,2 +1,22 @@
-## Throttling
+### Background
```

Comment:
```suggestion
### Network throttling in Elastic Synthetics
```
If this doc is to be consumed by customers, suggesting a clearer title to set the context right.